### PR TITLE
Feature/post blitz catalog

### PR DESCRIPTION
**Pull request by andrewpmiller**
## What this PR does:

- Adds a catalog page that is more suitable for post-blitz life
- Preview here: ~https://deploy-preview-248--thegymcms.netlify.com/static/take5/catalog~ https://deploy-preview-248--thegymcms.netlify.com/static/take5

### Important Notes

- Before submitting for review, turn off the debug view by uncommenting the conditional head/foot includes, and deleting the test include directives

### CODE REVIEW

**Comment by andrewpmiller** [comment]
@jgagne — for some reason the topics aren't visible when the page is rendering. 

The generated page source for that section shows the topic nav items:

```html
<div class="col-md-4">
<nav class="take5--tutorial-topics" aria-labelledby="tutorial-topics">
    <details class="take5--topics-menu" id="tutorial-topics">
    <summary aria-expanded="true/false" tabindex="0" role="button">
        <b role="heading">Topics</b>
    </summary>
    <div class="take5--topics-menu-list">
        <!-- Found 3 topic(s) with live content -->
        <ul>
        
        <li>
            <a href="#web-design-development"><b>Web Design & Development</b></a>
        </li>
        
        <li>
            <a href="#prototyping"><b>Prototyping</b></a>
        </li>
        
        <li>
            <a href="#career-skills"><b>Career Skills</b></a>
        </li>
        
        </ul>
    </div>
    </details>
</nav>
</div>
```
The rendered screen does not: 

image

Make sure to re-pull this branch before delving into it... I've updated the debug (commented) message to report the actual number of topics found)

**Update**

I think this is an issue with the media query magic you're doing. The topics appear in the mobile view (when you click the disclosure to expand the details).

*Note:* If you preview the page in mobile view and expose the list of topics, you can then increase the browser width and the topics appear correctly as the wider breakpoints are reached.

Mobile View:
image

**Comment by jgagne** [comment]
@andrewpmiller a quick patch for the topics section is to add the `open` attribute to the details element, which will display the section open on all screen widths. For example: `<details open class="take5--topics-menu" id="tutorial-topics">`. I’ll work on a proper fix. To be continued, but adding the `open` attribute for now will make the content accessible.

**Comment by andrewpmiller** [comment]
@jgagne - I think this is looking great. I’ll get it ready for review so we can get it staged and ready for production.

**Comment by andrewpmiller** [comment]
@mbifulco — this PR is ready for review whenever you've got a moment. ~Let me know if you run into conflicts, I'm happy to help sort them out.~ I believe I worked out the conflicts 😉

File: css/take5.css [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
...tination:0 0;scroll-snap-points-x:none;scroll-snap-points-y:none;scroll-snap-type:none;shape-image-threshold:0;shape-margin:0;shape-outside:none;tab-size:8;table-layout:auto;text-align:initial;text-align-last:auto;text-combine-upright:none;text-decoration-color:currentcolor;text-decoration-line:none;text-decoration-style:solid;text-emphasis-color:currentcolor;text-emphasis-position:over right;text-emphasis-style:none;text-indent:0;text-justify:auto;text-orientation:mixed;text-overflow:clip;text-rendering:auto;text-shadow:none;text-transform:none;text-underline-position:auto;top:auto;touch-action:auto;transform:none;transform-box:border-box;transform-origin:50% 50% 0;transform-style:flat;transition-delay:0s;transition-duration:0s;transition-property:all;transition-timing-function:ease;vertical-align:baseline;visibility:visible;white-space:normal;widows:2;width:auto;will-change:auto;word-break:normal;word-spacing:normal;word-wrap:normal;writing-mode:horizontal-tb;z-index:auto;-webkit-appearance:none;-moz-appearance:none;-ms-appearance:none;appearance:none;background:0 0;border-color:currentcolor;border-radius:0;border-style:none;border-width:medium;list-style:disc;margin:0;padding:0;
```
Comment (mbifulco) [inline css/take5.css:6]: 👀 what's going on here? CSS reset?

**Review by mbifulco — APPROVED** [review]
Looks like it's working from my end - pending the one question, let's get this thing into the real world 👍

File: css/take5.css (hunk 1 above)
Comment (andrewpmiller) [reply css/take5.css:6]: Uh... that’s a questionbest answered by @jgagne I think

Comment (jgagne) [reply css/take5.css:6]: This is (or was) a reset for bootstrap flavored structure (within the page template) outside of the include, carried over (that applied) from designing the UX Design landing page. However, if it no longer applies, we can omit.

**Pull request merged**